### DISCUSSION TITLE
Import data from Deezer

### DISCUSSION THREAD

**Discussion by Ectalite** [krateng/maloja#383] [Ideas]
Deezer provides a complete set of the song history in xslx format.
It's possible to convert the data in csv format to be able to import them, but it is not a neat way to do it.
It would be really easier to support this file format.

**Comment by Ectalite** [comment]
Here is an anonymized example of the file format:
[deezer-data_xxxxxxxxx.xlsx](https://github.com/user-attachments/files/17379371/deezer-data_xxxxxxxxx.xlsx)

**Comment by duckcantcode** [comment]
I could look into trying to implement this in a PR and/or my fork, but I'm really busy at the moment. Check out the Discord server at #226 if you want to talk to me.

**Comment by Ectalite** [comment]
Resolved with PR https://github.com/krateng/maloja/pull/421
